src/ecies.cpp

```diff
@@ -299,6 +299,12 @@ class ECIESEncryptionCtx::Impl : public ECIESImpl
 
 };
 
+size_t getEcPublicKeyOctetLength(const AsymmetricPrivateKey &privKey, openssl::EllipticCurvePointConversionForm form) {
+    size_t len = (privKey.getKeySize() + 7) / CHAR_BIT;
```

Comment:
in such complex computations, a more explicit variable name benefits the readability of the overall code
```suggestion
    size_t keySizeInBytes = (privKey.getKeySize() + 7) / 8; // getKeySize() returns bits, we round up to full bytes
```
Moreover, where does CHAR_BIT come from? The name is not sufficiently verbose to explain what is going on if I didn't already understand this. So maybe just put in 8 and write a comment?